### DISCUSSION TITLE
External Editor Startup Args Sharing

### DISCUSSION THREAD

**Discussion by Clazex** [bcssov/IronyModManager#176] [Show and tell]
Share startup args for an external editor to open a diff-like view for two files, which can be filled into Irony options directly.

e.g.
VS Code:  `-d {Left} {Right}`

**Comment by bcssov** [comment]
Beyond Compare
Exe ```BCompare.exe``` 
Args ```{Left} {Right} /fv="Text Compare"```

**Comment by novellus** [comment]
Meld
Exe `Meld.exe`
Args `{Left} {Right}`

**Comment by stephen-dahl** [comment]
Webstorm
Exe `webstorm64.exe`
Args `diff {Left} {Right}`
https://www.jetbrains.com/help/webstorm/command-line-differences-viewer.html
